### PR TITLE
fix: add legacy classname

### PR DESCRIPTION
**Pull request by sashuk**
[FX-4102]

### Description

The legacy `calendar-month` class name needs to be added to ease the migration to the new Calendar.

### How to test

- open the deployed Storybook and make sure that it is added

### Screenshots

Screenshot 2023-06-23 at 11 49 57

### Development checks

- [x] Add changeset according to [guidelines](https://github.com/toptal/picasso/blob/master/docs/contribution/changeset-guidelines.md) (if needed)
- [x] Read [CONTRIBUTING.md](https://github.com/toptal/picasso/blob/master/CONTRIBUTING.md) and [Component API principles](https://github.com/toptal/picasso/blob/master/docs/contribution/component-api.md)
- [N/A] Make sure that additions and changes on design follow [Toptal's BASE design](https://design.toptal.net/), and it's been already discussed with designers at #-base-core
- [N/A] Annotate all `props` in component with documentation
- [N/A] Create `examples` for component
- [x] Ensure that deployed demo has expected results and good examples
- [N/A] Ensure the changed/created components have not caused accessibility issues. [How to use accessibility plugin in storybook](https://github.com/toptal/picasso/blob/master/docs/contribution/accessibility.md).
- [x] Self reviewed
- [N/A] Covered with tests

> All **_development checks_** should be done and set checked to pass the
> **GitHub Bot: TODOLess** action

<details>
<summary>PR commands</summary>
<br />

List of available commands:

- `@toptal-bot run package:alpha-release` - Release alpha version
- `@toptal-anvil ping reviewers` - Ping FX team for review

</details>

<details>
<summary>PR Review Guidelines</summary>
<br />

#### When to approve? ✅

**You are OK** with merging this PR and

1. You have no extra requests.
2. You have optional requests.
   1. Add `nit:` to your comment. (ex. `nit: I'd rename this variable from makeCircle to getCircle`)

#### When to request changes? ❌

**You are not OK** with merging this PR because

1. Something is broken after the changes.
2. Acceptance criteria is not reached.
3. Code is dirty.

#### When to comment (neither ✅ nor ❌)

**You want your comments to be addressed** before merging this PR in cases like:

1. There are leftovers like unnecessary logs, comments, etc.
2. You have an opinionated comment regarding the code that requires a discussion.
3. You have questions.

#### How to handle the comments?

1. An owner of a comment is the only one who can resolve it.
2. An owner of a comment must resolve it when it's addressed.
3. A PR owner must reply with ✅ when a comment is addressed.

</details>

[FX-4102]: https://toptal-core.atlassian.net/browse/FX-4102?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by TomasSlama — APPROVED** [review]

**Comment by TomasSlama** [comment]
don't forget changeset 🙇

**Comment by sashuk** [comment]
@toptal-anvil ping reviewers

File: packages/picasso/src/Calendar/Calendar.tsx [hunk 1]
```diff
@@ -240,6 +240,8 @@ export const Calendar = forwardRef<HTMLDivElement, Props>(function Calendar(
               cell: classes.cell,
               vhidden: classes.vhidden,
             }}
+            // Keeping the legacy classname as it is heavily reused as a locator in tests
```
Comment (OleksandrNechai) [inline packages/picasso/src/Calendar/Calendar.tsx:243]:
```suggestion
            // Keeping the legacy classname as it is heavily used as a locator in tests
```

**Review by OleksandrNechai — APPROVED** [review]

File: packages/picasso/src/Calendar/Calendar.tsx (hunk 1 above)
Comment (sashuk) [reply packages/picasso/src/Calendar/Calendar.tsx]: @OleksandrNechai Updated, thanks!

**Pull request merged**